'Python/Python.h' file not found error when installing on OS X 10.9 Mavericks

Replacing with `#include <Python.h>` seems to fix the issue.

``` sh
running install
Checking .pth file support in /Library/Python/2.7/site-packages/
/usr/bin/python -E -c pass
TEST PASSED: /Library/Python/2.7/site-packages/ appears to support .pth files
running bdist_egg
running egg_info
writing requirements to src/watchdog.egg-info/requires.txt
writing src/watchdog.egg-info/PKG-INFO
writing top-level names to src/watchdog.egg-info/top_level.txt
writing dependency_links to src/watchdog.egg-info/dependency_links.txt
writing entry points to src/watchdog.egg-info/entry_points.txt
reading manifest file 'src/watchdog.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
no previously-included directories found matching 'docs/source/_themes/.git*'
writing manifest file 'src/watchdog.egg-info/SOURCES.txt'
installing library code to build/bdist.macosx-10.9-intel/egg
running install_lib
running build_py
running build_ext
building '_watchdog_fsevents' extension
cc -fno-strict-aliasing -fno-common -dynamic -arch x86_64 -arch i386 -g -Os -pipe -fno-common -fno-strict-aliasing -fwrapv -mno-fused-madd -DENABLE_DTRACE -DMACOSX -DNDEBUG -Wall -Wstrict-prototypes -Wshorten-64-to-32 -DNDEBUG -g -fwrapv -Os -Wall -Wstrict-prototypes -DENABLE_DTRACE -arch x86_64 -arch i386 -pipe -DWATCHDOG_VERSION_STRING="0.6.0" -DWATCHDOG_VERSION_MAJOR=0 -DWATCHDOG_VERSION_MINOR=6 -DWATCHDOG_VERSION_BUILD=0 -I/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c src/watchdog_fsevents.c -o build/temp.macosx-10.9-intel-2.7/src/watchdog_fsevents.o -std=c99 -pedantic -Wall -Wextra -fPIC
clang: warning: argument unused during compilation: '-mno-fused-madd'
src/watchdog_fsevents.c:22:10: fatal error: 'Python/Python.h' file not found
#include <Python/Python.h>
         ^
1 error generated.
error: command 'cc' failed with exit status 1
```

--- a/src/_watchdog_fsevents.h
+++ b/src/_watchdog_fsevents.h
@@ -21,7 +21,7 @@
 #ifndef _WATCHDOG_FSEVENTS_H_
 #define _WATCHDOG_FSEVENTS_H_ 1
 
-#include "Python.h"
+#include <Python.h>
 #include <stdlib.h>
 #include <signal.h>
 #include <limits.h>

--- a/src/watchdog_fsevents.c
+++ b/src/watchdog_fsevents.c
@@ -19,7 +19,7 @@
  */
 
 
-#include <Python/Python.h>
+#include <Python.h>
 #include <CoreFoundation/CoreFoundation.h>
 #include <CoreServices/CoreServices.h>
 #include <stdlib.h>

